Sync not working - 'null' article names (when they do have a name)

After upgrading to 0.7.0, I've come across the following error with all the articles in my list when syncing them after reading any of them on my Remarkable device:
 
```
[2025-02-05 19:10:15] Article 'BLABLA' has 1 pages and pocket id '4042842891'.
[2025-02-05 19:10:15] Executing command: [/usr/local/bin/rmapi_amd64, -ni, stat, /Pocket/BLABLA]
[2025-02-05 19:10:15] Executing command: [sed, /{/,$!d]
[2025-02-05 19:10:15] {
[2025-02-05 19:10:15]   "ID": "3483e62d-743f-46f6-9a99-94b73814cfae",
[2025-02-05 19:10:15]   "Name": "BLABLA",
[2025-02-05 19:10:15]   "Version": 0,
[2025-02-05 19:10:15]   "ModifiedClient": "2025-02-05T18:09:00Z",
[2025-02-05 19:10:15]   "Type": "DocumentType",
[2025-02-05 19:10:15]   "CurrentPage": 0,
[2025-02-05 19:10:15]   "Parent": "46b2c304-435e-4089-9cfc-3dbe6ffe083d"
[2025-02-05 19:10:15] }
[2025-02-05 19:10:15] null: Current page: 1, page count: 1.
[2025-02-05 19:10:15] Found 8 read article(s) on Remarkable.
[2025-02-05 19:10:15] (1/8) Marking 'null' as read on Pocket...
[2025-02-05 19:10:16] (1/8) Deleting 'null' from Remarkable...
[2025-02-05 19:10:16] Executing command: [/usr/local/bin/rmapi_amd64, -ni, rm, /Pocket/null]
[2025-02-05 19:10:16] ERROR: 2025/02/05 19:10:16 main.go:85: Error:  no matches for 'null'
```

Seems like something is off with the article names. Sorry for not notifying earlier on!

Thanks.

Jorge

PS. Do not know whether that happened before upgrading to 0.7.0... :'(

--- a/src/main/java/nl/carosi/remarkablepocket/model/Document.java
+++ b/src/main/java/nl/carosi/remarkablepocket/model/Document.java
@@ -4,5 +4,5 @@ import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 @JsonIgnoreProperties(ignoreUnknown = true)
-public record Document(
-    @JsonProperty("CurrentPage") int currentPage, @JsonProperty("VissibleName") String name) {}
+public record Document(@JsonProperty("CurrentPage") int currentPage, @JsonProperty("Name") String name) {
+}

